.circleci/config.yml

```diff
@@ -27,6 +27,8 @@ jobs:
           name: Clone rskj repo
           command: |
             git clone https://github.com/rsksmart/rskj.git
+            cd rskj
+            git checkout ddf2e391eeaaef1575cfc64d70ab065be73fa6fb
```

Comment:
this should be removed
```suggestion
```
circle was updated to search for a matching rskj branch to perform the build

Reply:
removed this part and added `rskj: active-block-reg` in the desc. Looks like it doesn't pick the rskj branch up.

Reply:
You should rename this branch to `active-block-reg` since CircleCI searches for a branch in rskj with a name equal to the branch it's running in powpeg. If it can't find it then it builds against `master`

Keep the comment `rskj: active-block-reg` in the description, that's for the pipeline